README.md

```diff
@@ -70,6 +70,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 - Checks headings have meaningful content:
   - They cannot be empty
   - They are reachable by assistive technologies (no `aria-hidden`)
+- Checks usage `accesskey` attribute in elements should be avoided
```

Comment:
I appreciate the PR I'm about to mention was being worked on at the same time as you were working on this, but could you have a read through #47 and change this accordingly? Basically, this line needs relocating into `features.md`. 😊

Reply:
I missed to read PR #47 
I'll update this PR then